src/PAModelTests/ReadTransformTests.cs

```diff
@@ -24,5 +24,22 @@ public void AfterReadNullChildren(string filename)
             msapp.ApplyAfterMsAppLoadTransforms(errorContainer);
             Assert.IsFalse(errorContainer.HasErrors);
         }
+
+        [DataTestMethod]
+        [DataRow("TestStepWithInvalidScreen.msapp")]
+        public void TestStepWithInvalidScreenAfterRead(string filename)
```

Comment:
Could probably make this more re-usable by introducing multiple parameters (filename, has errors, has warnings) which could share the test code with `AfterReadNullChildren` and possibly more...
```
        [DataTestMethod]
        [DataRow("GalleryTemplateNullChildren.msapp", false, false)]
        [DataRow("TestStepWithInvalidScreen.msapp", false, true)]
        public void ApplyAfterMsAppLoadTransforms_Test(string filename, bool hasErrors, bool hasWarnings)
        {
            var path = Path.Combine(Environment.CurrentDirectory, "Apps", filename);
            Assert.IsTrue(File.Exists(path));
            (var msapp, var errorContainer) = CanvasDocument.LoadFromMsapp(path);
            errorContainer.ThrowOnErrors();
            msapp.ApplyAfterMsAppLoadTransforms(errorContainer);
            Assert.AreEqual(errorContainer.HasErrors, hasErrors);
            Assert.AreEqual(errorContainer.HasWarnings, hasWarnings);
        }
```